kolibri/plugins/learn/assets/src/views/channels-page/index.vue

```diff
@@ -54,9 +51,7 @@
 
 <style lang="stylus" scoped>
 
-  .page-description
-    margin-top: 1em
-    margin-bottom: 1em
-    line-height: 1.5em
+  .grid
+    margin-top: 48px
```

Comment:
Let's reduce this margin-top to 16px, since we are also trying to make better use of space.

Reply:
16 looked a bit too small, but will shrink it.

(I was roughly lining them up with the cards in 'Recommended')

Reply:
I don't think it's necessary to line up.

Reply:
addressed